### ISSUE TITLE
Classeur for Academic Writing?

### ISSUE THREAD

**Issue by orschiro** [classeur/classeur#36]
Dear all, 

I'm interested what Classeur's stand is with respect to academic writing. 

Is Classeur intended to become a solid academic writing environment with good support for elements such as tables, figures, captions, references, title pages, rmarkdown (interactive figures and code) etc?

I'm asking out of interest because I'm not aware of any good Markdown Academic Writing environment. 

Looking forward to your opinions!

~ Robert

**Comment by TomBous** [comment]
Hi all,

I agree with @orschiro. Moreover, Classeur could be a knowledge management app, wich would be usefull for Academics and Businesses (both need to organize and structure knowledge). The Business part could allow Classeur to fund it's development, while the Academics part could be given for FREE, helping students and converting some of them, for when they become economically independent.

Btw, i noticed that Classeur doesn't support UML diagram as in StackEdit, is it going to be implemented ?

Thanks for your work !

**Comment by torbjornlager** [comment]
Hi, I would also appreciate a focus on academic writing. Seems you are already thinking in this direction. Classeur is not only good looking, Mathjax and export to Latex are great ideas and will certainly appeal to lots of CS people in academia. 

For businesses and non-CS academia you could consider doing Latex -> PDFs on your servers, Latex produces the best looking text in many people's eyes, but most users of Classeur won't bother installing a Latex system, and will therefore never know. 

Before adding more functionality to Classeur I believe reliability and stability must be increased. I have a long document that cannot be saved to PDF due to some bug, for example.  Let me know if you want me to send you something reproducible. 

I'm in academia but I'm a professor, so I don't mind paying. Students probably do, so a possibly reduced fee for the Enterprise version for universities and other schools might be a good idea. 

Thanks!

**Comment by benweet** [comment]
@torbjornlager yes, you should open a ticket for issues you have with PDF export (it's converted to latex and then to pdf on our servers with a custom pandoc template)

**Comment by torbjornlager** [comment]
Ok, I have opened a ticket.

The problem seems to be an invisible character. I don't know how it got
there. AFAICS, it's ascii 30.

Torbjörn

On Fri, Nov 27, 2015 at 12:39 AM, Benoit Schweblin <notifications@github.com

> wrote:
> 
> @torbjornlager https://github.com/torbjornlager yes, you should open a
> ticket for issues you have with PDF export (it's converted to latex and
> then to pdf on our servers with a custom pandoc template)
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/classeur/classeur/issues/36#issuecomment-160005715.

## 

Torbjörn Lager
Professor of General and Computational Linguistics
Department of Philosophy, Linguistics and Theory of Science
University of Gothenburg
Box 200, SE-405 30 Gothenburg, Sweden
Phone: +46317864962

**Comment by kenkangxgwe** [comment]
Hi, @benweet. Is there any way to access to the custom pandoc template that is used to produce PDFs? I didn't find it anywhere on Classeur. Pandoc options only allows me to change the depth of toc and style of highlights. It would be awesome, if I could change the latex template directly, so that I don't have to export to a latex file and use pandoc to convert it to pdf manually.

**Comment by benweet** [comment]
Hi,
That's not possible at the moment i'm afraid.
